webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -67,6 +67,7 @@ const poolAttributes = [
   { key: 'slotsPerAgent', label: 'Slots Per Agent' },
   { key: 'auxContainerCapacityPerAgent', label: 'Aux Containers Per Agent' },
   { key: 'schedulerType', label: 'Scheduler Type' },
+  { key: 'resourceManagerName', label: 'Resource Manager Name' },
```

Comment:
I don't know what this looks like in the code

but based on the screenshot, if someone specifies
```
name: firstname
metadata:
    name: nameinmetadata
```

Do we get a reasonable output / can we tell which name is from metadata?

Reply:
Good catch, it would just look like this, while the first row is always going to be the content from `resourceManagerName`. I'm not sure if we want some indicators for metadata, do we want another section for `resource manager metadata`? But the keys in metadata can be duplicated too. 
<img width="404" alt="Screenshot 2024-03-05 at 10 42 38 AM" src="https://github.com/determined-ai/determined/assets/40620519/fe7b8423-b0a5-4528-8bb6-b2f835c7f37c">

Another more direct way to display it is to just show metadata as JSON, but I don't think it looks nice
<img width="558" alt="Screenshot 2024-03-05 at 10 45 24 AM" src="https://github.com/determined-ai/determined/assets/40620519/5ce3d618-8825-4167-baf3-f7e784600c2e">

Reply:
```But the keys in metadata can be duplicated too``` Isn't it a map[string]string so the keys can't be duplicated?

Reply:
Oh right, I misunderstood